sphinx-ecosystem-news-january-2022.rst

```diff
@@ -0,0 +1,115 @@
+   By default, Sphinx transparently attempts to locate objects in external projects
+   if they are not found in the current one,
+   which is useful but also can become confusing in some cases.
+   The new :rst:role:`external` role, coupled with the
+   `intersphinx_disabled_reftypes <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_disabled_reftypes>`_
+   configuration introduced in Sphinx 4.3,
+   enables documentation writers to control
+   when intersphinx references should be used.
+
+   You can use the new ``:external:`` role as follows:
+
+   .. code-block:: rst
+
+      External reference: :external:py:class:`zipfile.ZipFile`.
+      External reference with constrained domain lookup: :external+python:py:class:`zipfile.ZipFile`.
+
+   See more information in :rst:role:`the official documentation <external>`.
+
+New asynchronous methods to load JavaScript files
+   The method :py:meth:`sphinx:sphinx.application.Sphinx.add_js_file`
+   now accepts a new ``loading_method`` parameter that can either be ``"async"`` or ``"defer"``,
+   which results in the script being loaded with the
+   `async <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async>`_ or
+   `defer <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer>`_
+   HTML attributes respectively.
+
+Proper error messages when autosummary does not find a package
+   Sphinx ships the :std:doc:`autosummary <sphinx:usage/extensions/autosummary>` extension
+   to automatically generate API pages of a Python library.
+   However, autosummary used to raise misleading error messages if the target library failed to import,
+   which resulted in user frustration.
+   This problem has now been fixed and autosummary properly points out the reason of the ``ImportError``.
+
+You can read the `complete Sphinx 4.4.0 release
+notes <https://www.sphinx-doc.org/en/master/changes.html#release-4-4-0-released-jan-17-2022>`_ online.
+
+**pydata-sphinx-theme**, a Sphinx theme based on Bootstrap CSS developed by the PyData community,
+`got a new 0.8 release <https://github.com/pydata/pydata-sphinx-theme/releases/tag/v0.8.0>`_
+on January 15th. The Read the Docs team collaborated with the developers
+to make some of the new features behave correctly in our platform, in particular
+`the custom version
+switcher <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#add-a-dropdown-to-switch-between-docs-versions>`_.
+As `Chris Holdgraf summarizes in this Twitter
+thread <https://twitter.com/choldgraf/status/1482435411301449729>`_,
+the new version brings other interesting additions,
+like `better depth control of the left
+sidebar <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#navigation-depth-and-collapsing-of-the-sidebar>`_
+and `custom SVG
+icons <https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/configuring.html#local-image-icons>`_.
+
+Extensions
```

Comment:
These feel like good tweet content, and we reframe the blog post as annotated sphinx release notes?

Reply:
Not sure if you're advocating for removing this from the blog post and putting it in a Twitter thread, or just restructuring it a bit?

Reply:
Restructuring the post, but also using this content as a tweet. If we get better at tracking extensions, I think it's great twitter content, especially with the GIF's.